content/utilities/mini-map/plugin.js

```diff
@@ -158,7 +158,9 @@ class Plugin {
               data.push({
                 x: i,
                 y: j,
-                type: df.spaceTypeFromPerlin(df.getPerlin({x: i, y: j}))
+                type: df.entityStore.spaceTypeFromPerlin(
```

Comment:
```suggestion
                type: df.spaceTypeFromPerlin(
```
Looks like `spaceTypeFromPerlin` is still on the `df` object